shared/core/types/proto/challenge.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package core;
+
+option go_package = "github.com/pokt-network/pocket/shared/core/types";
+
+import "relay.proto";
+
+message Challenge {
```

Comment:
Can you add a `TECHDEBT: These structures were copied as placeholders from v0 and need to be updated to reflect changes in v1; <insert_link_to_v0_ref_here>`